tests/ttnn/unit_tests/operations/test_eltwise_softplus_inf.py

```diff
@@ -0,0 +1,83 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+from loguru import logger
+import random
+import pytest
+import torch
+import ttnn
+import traceback
+
+from tests.ttnn.python_api_testing.sweep_tests import ttnn_ops
+from tests.tt_eager.python_api_testing.sweep_tests.comparison_funcs import comp_pcc
+from models.utility_functions import skip_for_grayskull
+
+
+def run_eltwise_softplus_tests(
+    input_shape,
+    dtype,
+    dlayout,
+    in_mem_config,
+    output_mem_config,
+    beta,
+    threshold,
+    data_seed,
+    device,
+):
+    torch.manual_seed(data_seed)
+    x = torch.Tensor(size=input_shape[0]).uniform_(-100, 100)
+
+    try:
+        # get ref result
+        ref_value = torch.nn.functional.softplus(x, beta=beta, threshold=threshold)
+
+        x = ttnn_ops.setup_ttnn_tensor(x, device, dlayout[0], in_mem_config[0], dtype[0])
+        tt_result = ttnn.softplus(x, beta=beta, threshold=threshold, memory_config=output_mem_config)
+
+        tt_result = ttnn_ops.ttnn_tensor_to_torch(tt_result, output_mem_config)
+
+    except Exception as e:
+        logger.warning(f"Test execution crashed: {e}")
+        print(traceback.format_exc())
+        raise e
+
+    assert len(tt_result.shape) == len(ref_value.shape)
+    assert tt_result.shape == ref_value.shape
+
+    # print(f"PyTorch: {ref_value[0, 0, 0:9, 0:9]}")
+    # print(f"TT: {tt_result[0, 0, 0:9, 0:9]}")
```

Comment:
Do we need this commented code?

Reply:
not really. I left it there in case the ticket owner needed to check. will remove it